test/parallel/test-timers.js

```diff
@@ -68,14 +68,14 @@ inputs.forEach(function(value, index) {
 // All values in inputs array coerce to 1 ms. Therefore, they should all run
 // before a timer set here for 2 ms.
 
-setTimeout(common.mustCall(function() {
+setTimeout(common.mustCall(() => {
   // assert that all other timers have run
-  inputs.forEach(function(value, index) {
+  inputs.forEach((value, index) => {
     assert(timeouts[index]);
     assert(intervals[index]);
   });
 }), 2);
 
 // Test 10 ms timeout separately.
 setTimeout(common.mustCall(), 10);
-setInterval(common.mustCall(function() { clearInterval(this); }), 10);
+setInterval(common.mustCall(() => { clearInterval(this); }), 10);
```

Comment:
This makes `this` lexical. Is it OK here?

Reply:
@vsemozhetbyt Thank you for reviewing. I missed it. I will fix it.

Reply:
Fixed.